src/main/scala/gestalt/dotty/Expander.scala

```diff
@@ -67,7 +76,8 @@ object Expander {
   /** Expand def macros */
   def expandDefMacro(tree: tpd.Tree)(implicit ctx: Context): untpd.Tree = tree match {
     case ExtractApply(Select(obj, method), targs, argss) =>
-      val className = obj.symbol.info.classSymbol.fullName + "$inline$"
+      val objType = obj.symbol.info.resultType
+      val className = javaClassName(objType.classSymbol) + "$inline$"
```

Comment:
Another reliable way I see is to first get the symbol of the select, it must refer to the macro method. Then the parent symbol of the method is the class symbol we need.